data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -170,7 +170,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If our fate is death then so be it. Until that time comes, my people will fight and I will lead them to fight!"
+            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If that is our fate is death, so be it. Until that time comes, I will lead my people to fight for our freedom!"
```

Comment:
You forgot to remove the "is death" after fate.

Reply:
Fixed it.